### PR TITLE
Temporary fix for azure deployment until we support test offer installs

### PR DESCRIPTION
**Pull request by mbelur**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,6 +47,7 @@ kubeconfig-e2e.yml
 
 # Ignore the downloaded CA certificate for tls-rancher
 *ca-additional.pem
+*.pem
```
Comment (guangyee) [inline .gitignore:50]: `*.pem` should cover all the certs so line 49 may no needed

File: terraform/aws/variables.tf [hunk 2]
```diff
@@ -107,13 +107,13 @@ variable "deploy_payg_adapter" {
 variable "rancher_version"{
   type = string
   description = "the version of rancher. Must be compatable with the cert-manager/eks version"
-  default = "2.7.5"
+  default = "2.7.7"
```
Comment (guangyee) [inline terraform/aws/variables.tf:110]: I am rolling a 2.7.7 right now to test upgrade scenario. But the production release is expected to be 2.7.8, or 2.8.0 if it comes out in time.

**Review by guangyee — APPROVED** [review]
Just a minor suggestion, looks good otherwise.

**Comment by guangyee** [comment]
For the image repos and tags, you can parse them from CNAB publisher if you want.

https://gitlab.suse.de/public-cloud/rancher-azure-cnab-publisher/-/blob/main/container_images?ref_type=heads

**Comment by mbelur** [comment]
> Just a minor suggestion, looks good otherwise.

updated

**Comment by mbelur** [comment]
> For the image repos and tags, you can parse them from CNAB publisher if you want.
> 
> https://gitlab.suse.de/public-cloud/rancher-azure-cnab-publisher/-/blob/main/container_images?ref_type=heads

since the registry.suse.de image are not accessible from outside, temporarily using docker registry until the implementation for test offer installs is available.

**Pull request merged**